lib/core/text/abstract_text.nit

```diff
@@ -1604,13 +1604,28 @@ redef class Int
 	`}
 
 	# return displayable int in base base and signed
```

Comment:
This doc needs some love... I would suggest:
~~~
String representation of `self` in the given `base`
~~~